docs/guides-agent-app.md

```diff
@@ -0,0 +1,485 @@
+
+You can do this either directly through the UI or by running:
+
+```
+dao exec <name of dao B> <voting app address of dao B> vote 0 true true --environment aragon:rinkeby --apm.ipfs.rpc https://ipfs.eth.aragon.network/ipfs/
+```
+
+On the UI, the vote will look something like this.
+
+<p align="center">
+   <img width="800" src="/docs/assets/agent-guide/agent-8.png">
+</p>
+
+Once you've voted *yes*, switch over to the Token Manager app (click on *Tokens* in the left sidebar).
+
+<p align="center">
+   <img width="800" src="/docs/assets/agent-guide/agent-9.png">
+</p>
+
+You should see that you've successfully added another token holder (your Agent app)!
+
+### **3. Create a vote in B to add a third entity**
+
+As in step 2, we'll run `dao exec` again, except this time the first argument to `mint` will be the address of the third entity we want to add to B.
+
+```
+dao exec <name of dao B> <token manager app address of dao B> mint <third entity's address> 1000000000000000000 --environment aragon:rinkeby --apm.ipfs.rpc https://ipfs.eth.aragon.network/ipfs/
+```
+
+Running the above will create an open vote in B. Again we'll need to vote *yes* to confirm the minting.
+
+<p align="center">
+   <img width="800" src="/docs/assets/agent-guide/agent-10b.png">
+</p>
+
+As before, you can either do this through the UI or run the same command we ran at the end of step 2 with one small modification: 
+
+This time the first argument to `vote` will be a `1` and not a `0`, since the id of this new vote is 1. Remember that vote ids start at zero and increment by one each time a vote is created.
+
+If you go to the Voting app after you've voted, you'll see that the *yes* vote is only 50%.
+
+<p align="center">
+   <img width="800" src="/docs/assets/agent-guide/agent-10.png">
+</p>
+
+That's because only 1 / 2 tokenholders have voted. The other tokenholder (A's Agent app) still has to vote.
+
+
+### **4. Use A's Agent app to take part in B's vote**
+
+In order to close and enact the vote, we'll use A's Agent app to vote yes to adding a third entity to B.
+
+But before we do this, we need to introduce the `dao act` command.
+
+According to the documentation:
+
+>`dao act` provides some syntax sugar over `dao exec` for executing actions using Agent app instances in a DAO.
+
+In plain english, `dao act` acts like `dao exec` except the first argument to `dao act` is an agent app address rather than a DAO address.
+
+Another minor but important difference is that we need to pass in the full signature of the method we wish to execute.
+
+For example, if we want to execute the `vote` method of a Voting app we need to pass in `vote(unint256,bool,bool)`.
+
+Now that you have a high-level understanding of `dao act`, you're ready to run the following command:
+
+```
+dao act <agent app address of dao A> <voting app address of dao B>  "vote(uint256,bool,bool)" 1 true true  --environment aragon:rinkeby --apm.ipfs.rpc https://ipfs.eth.aragon.network/ipfs/
```

Comment:
In this particular case it is pretty important to explain all the parameters of `dao act` as it is the way users will be interacting with the Agent app